packages/anasazi/tpetra/example/CustomStatusTest/LOBPCGCustomStatusTest.cpp

```diff
@@ -47,7 +47,7 @@
 #include "AnasaziTypes.hpp"
 
 #include "Teuchos_GlobalMPISession.hpp"
```

Comment:
Remove this header?

Reply:
@hkthorn Yup!  (Is it OK if I go back later for this?)